web/client/components/styleeditor/FilterBuilder.jsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import GroupField from '../data/query/GroupField';
+import Popover from '../misc/Popover';
+import { Glyphicon, Button as ButtonRB } from 'react-bootstrap';
+import tooltip from '../misc/enhancers/tooltip';
+
+const Button = tooltip(ButtonRB);
+
+const updateFilterField = (field = {}, action = {}) => {
+    const newField = {
+        ...field,
+        [action.fieldName]: action.fieldValue,
+        type: action.fieldType,
+        fieldOptions: {
+            ...field.fieldOptions,
+            currentPage: action.fieldOptions.currentPage === undefined ? 1 : action.fieldOptions.currentPage
+        }
+    };
+    if (action.fieldName === 'attribute') {
+        return {
+            ...newField,
+            value: action.fieldType === 'string' ? '' : null,
+            operator: ''
+        };
+    }
+    if (action.fieldName === 'operator') {
+        return {
+            ...newField,
+            value: null
+        };
+    }
+    return newField;
+};
+
+const FilterBuilder = ({
+    filterObj = {
+        groupFields: [{ id: 1, logic: 'OR', index: 0 }]
+    },
+    attributes = [],
+    groupLevels = 0,
+    onChange = () => {}
+}) => {
+    const { groupFields, filterFields } = filterObj;
+    return (
+        <div className="ms-style-rule-filter">
+            <GroupField
+                attributes={attributes}
+                filterFields={filterFields}
+                groupFields={groupFields}
+                autocompleteEnabled={false}
+                groupLevels={groupLevels}
+                withContainer={false}
+                listOperators={['=']}
+                stringOperators={['=', '<>', 'like', 'isNull']}
+                booleanOperators={['=']}
+                defaultOperators={['=', '>', '<', '>=', '<=', '<>']}
```

Comment:
@offtherailz 

> I think because of this, there is no "not between operator (><), for instance in dates (and other operators). I'm trying to guess, you excluded it for horizontal space issues, isn't it?

Current style filter supports only a minimum set of operator and it will support only string and number at the moment. This is the reason for those changes in the group filter to customize operator. 

> But in filter filed you changed something on that part. Is this correct ? did you fixed something else?

In addition to those new props I replaced all Row and Cols with a `display: flex` system